packages/amplication-client/src/Project/ArchitectureConsole/types.ts

```diff
@@ -13,6 +13,7 @@ export type NodePayload<T> = {
   height?: number;
   originalParentNode?: string;
   isCurrentDropTarget?: boolean;
+  highlight: boolean;
```

Comment:
make this param options
```suggestion
  highlight?: boolean;
```